test/test_path_finding.jl

```diff
@@ -41,6 +41,11 @@
     p = Path([1, 2], [10.0])
     @test p == Path([1, 2], [10.0])
     @test p == Path([2, 1], [10.0])
+
+    # hash
+    p1 = Path([1, 2], [10.0])
+    p2 = Path([2, 1], [10.0])
+    @test hash(p1) == hash(p2)
```

Comment:
Maybe also add a symmetric test case?
```suggestion
    @test hash(p1) == hash(p2)
    p1 = Path([1, 2, 1], [10.0, 20.0])
    p2 = Path([1, 2, 1], [20.0, 10.0])
    @test hash(p1) == hash(p2)
```